notebooks/Project and Developer Similarity.ipynb

```diff
@@ -0,0 +1,625 @@
+{
```

Comment:
<p>Wow,  did not know in Jupiter one can define a function that does <code>!echo 'something'</code>!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mloncode/devfest2019-workshop/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Yes, it's also very handy to be able to use that in <code>multiprocessing</code> :)</p>